cloud_chemistry/prototype/config/model.json

```
@@ -0,0 +1,20 @@
+{
+  "version": "1.0.0",
+  "name": "Hard-coded Cloud Chemistry",
+  "model": [
+    {
+      "type": "two-moment modal",
+      "distribution": "log normal",
+      "geometric mean diameter [m]": 1.0e-8,
+      "geomoetric standard deviation": 2.4
+    },
+    {
+      "type": "bin sectional modal",
+      "radius": 0.1e-6
+    },
+    {
+      "type": "bin sectional modal",
+      "radius": 0.2e-6
+    }
```

```suggestion
      "type": "two-moment modal",
      "distribution": "log normal",
      "geometric mean diameter [m]": 1.0e-8,
      "geomoetric standard deviation": 2.4,
      "phases": [ "aqueous" ]
    },
    {
      "type": "bin sectional modal",
      "radius": 0.1e-6,
      "phases": [ "aqueous" ]
    },
    {
      "type": "bin sectional modal",
      "radius": 0.2e-6,
      "phases": [ "aqueous" ]
    }
```
The phases included in each bin or mode I think should also be included in the configuration (either here or somewhere else)

---

Done!